ViAn/GUI/mainwindow.cpp

```diff
@@ -336,17 +337,25 @@ void MainWindow::on_bookmark_button_clicked() {
         // Get current project.
         item = ui->project_tree->selectedItems().first();
         my_project = (MyQTreeWidgetItem*)get_project_from_object(item);
-        QDir proj_path = fileHandler->get_dir(my_project->id);
         // Add bookmarks-folder to the project-folder.
-        proj_path.mkpath(proj_path.absoluteFilePath("Bookmarks"));
-        proj_path.cd(proj_path.absoluteFilePath("Bookmarks"));
-
+        Project* proj = fileHandler->get_project(my_project->id);
+        QDir dir = fileHandler->get_dir(proj->bookmark_dir);
         // Export the current frame in the bookmarks-folder.
         // The names of the stored files will have increasing numbers.
+
```

Comment:
This comment is connected to the code a couple lines below:
`std::string file_name = std::to_string(bookmark_view->get_num_bookmarks());`